src/server/js/server.js

```diff
@@ -1,4 +1,6 @@
 require('babel-core/register');
+require('dotenv').config();
```

Comment:
this `dotenv` call automagically adds the contents of the `.env` file in the application root as node environment variables.